src/storage.ts

```diff
@@ -2,7 +2,15 @@
  * Contains functions that interact with the backing storage.
  */
 
+import {IncomingCloudflareProperties} from 'worktop/request';
```

Comment:
Nit: `import type {}` when you're using it only as a type.
```suggestion
import type {IncomingCloudflareProperties} from 'worktop/request';
```